roles/crawler/tasks/configure-databases.yml

```diff
@@ -99,6 +110,12 @@
       lc_ctype: en_US.UTF-8
       template: "{{ fpsd_database_psql_env.PGDATABASE }}"
 
-  when: raw_schema_result|changed or raw_schema_tables_result|changed
+  - name: Setup TABLEFUNC extension.
```

Comment:
This task shouldn't need to run. The TABLEFUNC extension should come along with the template.